src/routes/welcome/gettingStarted/connect.tsx

```diff
@@ -0,0 +1,139 @@
+import { Spacer, Text } from "@arconnect/components";
+import twitterLogo from "url:/assets/setup/twitter-logo.svg";
+import discordLogo from "url:/assets/setup/discord-logo.svg";
+import infoLogo from "url:/assets/setup/info-logo.svg";
+import githubLogo from "url:/assets/setup/github-logo.svg";
+import Paragraph from "~components/Paragraph";
+import browser from "webextension-polyfill";
+import styled from "styled-components";
+import { useEffect } from "react";
+import { PageType, trackPage } from "~utils/analytics";
+
+export default function Connect() {
+  // Segment
+  useEffect(() => {
+    trackPage(PageType.SETUP_CONNECT);
+  }, []);
+
+  return (
+    <Wrapper>
+      <div>
+        <Text heading>{browser.i18n.getMessage("connect_with_us_title")}</Text>
+        <Paragraph>{browser.i18n.getMessage("connect_paragraph")}</Paragraph>
+        <Paragraph>{browser.i18n.getMessage("connect_paragraph_2")}</Paragraph>
+      </div>
+      <Content>
+        <Item
```

Comment:
Could you please add the usual animations, so the users can "feel" feedback when hovering/pressing these logos. (by usual, I mean, ~.8 opacity on hover, ~.925 scale on `:active`).